File: AlarmApp3/AlarmApp3/ViewController.m
//
//  ViewController.m
//  AlarmApp3
//
//  Created by 이상학 on 2015. 9. 1..
//  Copyright (c) 2015년 ryan. All rights reserved.
//

#import "ViewController.h"
#import "MainViewController.h"

@interface ViewController ()

@end

@implementation ViewController
@synthesize infoButton;
@synthesize mainViewController;

- (void)viewDidLoad { // 새로 정의
    MainViewController *viewController = [self.storyboard
                                          instantiateViewControllerWithIdentifier:@"MainViewController"];
    mainViewController = viewController; // 기존의 뷰말고 새로 만든 mainViewController  쓰는듯
    [self.view insertSubview:viewController.view belowSubview:infoButton]; // infoButton뒤로 mainViewController 넣음
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Vibration/Vibration/ViewController.swift
//
//  ViewController.swift
//  Vibration
//
//  Created by 이상학 on 2015. 9. 26..
//  Copyright © 2015년 ryan. All rights reserved.
//

import UIKit
import AVFoundation
import AudioToolbox

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func buttonTapped(sender: AnyObject) { // 진동 버튼 클릭하면 실해
        AudioServicesPlaySystemSound(4095)
        
    }

}

